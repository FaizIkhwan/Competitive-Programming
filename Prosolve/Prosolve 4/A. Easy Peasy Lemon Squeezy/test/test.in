3
1 2 1
1 10 2
99 100 2