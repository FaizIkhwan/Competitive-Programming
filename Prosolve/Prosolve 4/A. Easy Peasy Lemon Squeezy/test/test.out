1 2 
1 3 5 7 9 
99 
